docker/container.go

```diff
@@ -96,6 +97,51 @@ func name(names []string) string {
 	return current[1:]
 }
 
+// Rebuild will delete the container and create a new one with the current configuration
+func (c *Container) Rebuild(imageName string) (*dockerclient.APIContainers, error) {
```

Comment:
I think "build" is the wrong terminology here. We build images, but we create containers. I think it should be `Recreate()`